src/main/java/com/xeeva/catalog/CostCenterPage.java

```diff
@@ -1,532 +1,583 @@
+						statusFlag = true;			
+						break;
+					}
+				}
+			}
+
+			break;
+
+		case "headerlevel": 
+			for(int i=0;i<readSelects.size();i++){
+				String ExpectedCC = new Select(readSelects.get(i)).getFirstSelectedOption().getText();
+				if(ExpectedCC.equalsIgnoreCase(CCValue)){
+					statusFlag =true;
+				}
+			}
+			break;
+
+		case "copyitem":
+			for(WebElement input :readSelects){
+				if(new Select(input).getFirstSelectedOption().getText().contains(CCValue)){
+					String randomID =input.getAttribute("id").replaceAll("\\D+", "");
+					WebElement element = driver.findElement(By.xpath(".//*[@id='trCCRow_"+randomID+"']/td[9]"));
+					driver.executeJavaScript("arguments[0].click();", element);
+					if(txtShowFullComment.isDisplayed()){
+						TestReporter.assertTrue(false, "Copied Item having Internal Comments");
+					}else{
+						TestReporter.assertTrue(true, "Copied Item is not having Internal Comments");
+					}
+				}
+			}
+			break;
+
+		case "quantity": 
+			// Change Quantity - Line Level record
+			changeQuantity(QuantityValue,lstSave.get(0));
+			break;
+		case "savecart": 
+			// Verify Save Cart 
+			changeQuantity(QuantityValue,btnSaveCartCC);
+			break;
+
+		case "delete":
+			// Perform Delete 
+			int sizeBeofreDelete =lstQty.size();
+			driver.executeJavaScript("arguments[0].click();", readDeleteIcons.get(0));
+			//Handle Alert if present
+			if(AlertHandler.isAlertPresent(driver, 6)){
+				AlertHandler.handleAlert(driver, 6);
+			}
+			// Waiting for quantity table 
+			Sleeper.sleep(2000);
+			int sizeAfterDelete =lstQty.size();
+			TestReporter.assertTrue(sizeAfterDelete<sizeBeofreDelete, "Item Deleted Successfully!!");
+			break;
+		case "edititem" :
+			//Read item description here
+			for(WebElement input :readText){
+				if(input.getText().equalsIgnoreCase(ItemDescription)){
+					TestReporter.log("ItemDescription is :"+ItemDescription);
+					if(lstEditItemGrid.get(lstEditItemGrid.size()-1).isEnabled()){
+						lstEditItemGrid.get(lstEditItemGrid.size()-1).click();
+						TestReporter.assertTrue(true, "Clicked Edit Link on Non-Price Agreement Item");
+						break;
+					}
+				}
+			}
+			break;
+		default : System.out.println();
+
+		}
+		return statusFlag;
+	}
+
+
+
+	/**
+	 * @summary: Method to get the date as per the daysout value.
+	 * @param daysOut
+	 * @return
+	 */
+	public String getDate(String daysOut){
+		DateTimeConversion dteTimeCon = new DateTimeConversion();
+		String setDaysOut = dteTimeCon.getDaysOut(daysOut, "MM/dd/yyyy").toString();
+		TestReporter.logStep("No. of days-out : " + daysOut);
+		//System.out.println("Date : "+setDaysOut);
+		return setDaysOut;
+	}
+
+	/**
+	 * @summary Method to change Cost Center
+	 * @author  Lalitha Banda
+	 * @date    16/09/16
+	 */
+	public void change_CC(String changeType,String CC){
+		// after cart check out, application taking time to load cost center page 
+		lblCC.syncVisible(30, false);
+		lblCC.isDisplayed();
```

Comment:
If you are not doing anything with the return value of isDisplayed(), please remove this line